src/garage/torch/algos/ddpg.py

```diff
@@ -69,21 +75,20 @@ def __init__(self,
                  rollout_batch_size=1,
                  exploration_strategy=None,
                  target_update_tau=0.01,
-                 policy_lr=1e-4,
-                 qf_lr=1e-3,
                  discount=0.99,
                  policy_weight_decay=0,
                  qf_weight_decay=0,
-                 optimizer=torch.optim.Adam,
+                 policy_optimizer=torch.optim.Adam,
+                 qf_optimizer=torch.optim.Adam,
+                 policy_lr=_Default(1e-4),
+                 qf_lr=_Default(1e-3),
                  clip_pos_returns=False,
                  clip_return=np.inf,
                  max_action=None,
                  reward_scale=1.,
                  smooth_return=True):
         action_bound = env_spec.action_space.high
         self.tau = target_update_tau
-        self.policy_lr = policy_lr
```

Comment:
while you are here, please make all these members _private

Reply:
Done.